src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -324,7 +425,28 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 			}
 
 		});
-		return rows;
+
+		let formattedRows: { [key: string]: string }[] = [];
+		let commonRows: { [key: string]: string }[] = [];
+		for (const [key, value] of Object.entries(rows)) {
+			if (value.primary && value.secondary && value.primary['text'] === value.secondary['title']) {
+				commonRows.push(value);
+			}
+			else {
+				formattedRows.push(value);
+			}
+		}
+
+		if (commonRows.length > 0) {
+			let commonRow = {};
+			commonRow['name'] = '';
```

Comment:
Where are all these keys coming from? In general we should avoid "magic" property names like this and instead define a type to use - this helps guarantee some level of type checking at least.